docs/usage/shoot_status.md

```diff
@@ -51,20 +51,43 @@ Even if the `failurePolicy` is set to `Ignore`, high timeouts (`>15s`) can lead
 the overall request might still fail as there is overhead in communication with the API server and potential other webhooks.
 Generally, it's [best pratice](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts) to specify low timeouts in WebhookConfigs.
-Also, it's [best practice](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#avoiding-operating-on-the-kube-system-namespace)
-to exclude the `kube-system` namespace from webhooks to avoid blocking critical operations on system components of the cluster.
-Shoot owners can do so by adding a `namespaceSelector` similar to this one to their webhook configurations:
+
+As an effort to correct this common problem, the webhook remediator has been created. This feature simply checks whether a customer's webhook matches a set of rules, described [here](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/matchers/matcher.go#L66-L180). If at least one of the rule matches, it will change the constraints (`HibernationPossible` & `MaintenancePreconditionsSatisfied`) statues.
+
+In most cases, you can avoid this by simply excluding the kube-system namespace from your webhook.
 ```yaml
-namespaceSelector:
-  matchExpressions:
-  - key: gardener.cloud/purpose
-    operator: NotIn
-    values:
-    - kube-system
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+webhooks:
+  - name: my-webhook.example.com
+    namespaceSelector:
+      matchExpressions:
+      - key: gardener.cloud/purpose
+        operator: NotIn
+        values:
+          - kube-system
+    rules:
+      - operations: ["*"]
+        apiGroups: [""]
+        apiVersions: ["v1"]
+        resources: ["pods"]
+        scope: "Namespaced"
 ```
 
-If the Shoot still has webhooks with either `failurePolicy={Fail,nil}` or `failurePolicy=Ignore && timeoutSeconds>15` that act on [critical resources](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/matchers/matcher.go#L60) in the `kube-system` namespace, Gardener will set the `HibernationPossible` to `False` indicating, that the Shoot can probably not be woken up again after hibernation without manual intervention of the Gardener Operator.
-`gardener-apiserver` will prevent any Shoot with the `HibernationPossible` constraint set to `False` from being hibernated, that is via manual hibernation as well as scheduled hibernation.
+However, some other resources (mostly without namespaces) might still trigger the remediator, namely:
+- endpoints
+- nodes
+- podsecuritypolicies
+- clusterroles
+- clusterrolebindings
+- customresourcedefinitions
+- apiservices
+- certificatesigningrequests
+- priorityclasses
+
+In these cases, please make sure that your `rules` don't overlap with one of those resources (and their subresources).
```

Comment:
I don't understand this sentence - what does it mean/what do I need to do when I have a webhook for `ClusterRole`s, for example?

Reply:
Just changed the sentence as it was indeed not clear enough. Basically, my point was : if one of the non-namespaced resources triggers the remediator, simply remove that resource from the webhook's rule.

Reply:
OK, but what if I can't remove it because my software relies on this webhook to be present?

Reply:
Well AFAIK, that's how the webhook remediator works (I guess that's a limitation)... I don't think there is another way around it. Are you aware of another way to work around that for non-namespaced resources ?

Reply:
In such (rather rare/special, I assume) cases, end-users should probably label their webhooks with `remediation.webhook.shoot.gardener.cloud/exclude=true` as explained in the document as well to not be bothered by it. However, we should make clear that this should only be used in those cases and not to "quickly silence/stop the auto-remediation because it bothers me".

Reply:
Good point! I just revamped the last section to make it clearer.